src/modules/getRequests.js

```diff
@@ -4,7 +4,8 @@ import createHeaders from "./headers.js";
 const getRequests = async (dispatch, category) => {
   try {
     const response = await axios.get("/requests", {
-      params: { category: category.activeCategory },
+      category: category,
```

Comment:
With this one I mostly experimented. It worked before and still does. Difference is that this new solution hides the query param in the url if that is desirable